app/forms/journeys/additional_payments_for_teaching/reminders/email_verification_form.rb

```diff
@@ -47,6 +47,22 @@ def sent_one_time_password_at?
         rescue Date::Error
           false
         end
+
+        def load_current_value(attribute)
```

Comment:
Have a ticket to sort this https://dfedigital.atlassian.net.mcas.ms/browse/CAPT-1719